### PR TITLE
#371 - handle SERIAL datatype properly

### PR DESCRIPTION
**Pull request by osheroff**
@zendesk/rules

### CODE REVIEW

**Comment by dadah89** [comment]
```
SERIAL is an alias for BIGINT UNSIGNED NOT NULL AUTO_INCREMENT UNIQUE.

SERIAL DEFAULT VALUE in the definition of an integer column is an alias for NOT NULL AUTO_INCREMENT UNIQUE.
```
I see `NOT NULL` matches the `nullability` definition, but from `column_options` the only think we seem to care is `primary_key`?

**Comment by osheroff** [comment]
that's correct.  but I think we roughly match the mysql parser by accepting all column options (and then ignoring the ones we don't care about).

File: Makefile [hunk 1]
```diff
@@ -17,6 +17,9 @@ ANTLR_IMPORTS=src/main/antlr4/imports
 ANTLR_DIR=target/generated-sources/src/main/antlr4/com/zendesk/maxwell/schema/ddl
 ANTLR_OUTPUT=$(ANTLR_DIR)/mysqlBaseListener.java $(ANTLR_DIR)/mysqlLexer.java $(ANTLR_DIR)/mysqlListener.java $(ANTLR_DIR)/mysqlParser.java
 
+$(ANTLR_IMPORTS)/mysql_literal_tokens.g4: $(ANTLR_IMPORTS)/generate_tokens.rb
```
Comment (dadah89) [inline Makefile]: this step was executed manually before this change?

**Comment by dadah89** [comment]
lgtm 👍

File: Makefile (hunk 1 above)
Comment (osheroff) [inline Makefile]: yeah

**Pull request merged**